### PR TITLE
ML2-196 Add IE notice for managed lending 

### PR DESCRIPTION
**Pull request by ryan-ludwig**

### CODE REVIEW

**Review by mcstover — APPROVED** [review]
Def a candidate for a global component...

**Review by emuvente — COMMENTED** [review]
If their browser in unsupported, this message might not be shown. If there's a javascript feature we're using that isn't supported by their version of IE, then the javascript won't parse and this won't get executed.
Alternatively, we could check the user agent string from the request on the server, allowing the message to be rendered on the server, and then it doesn't matter if the javascript works in their browser or not.

**Comment by ryan-ludwig** [comment]
That's a really good point @emuvente

File: src/main.js [hunk 1]
```diff
@@ -65,6 +66,7 @@ export default function createApp({
 	// Provide locale to all components
 	// TODO: use this to set locale in VueI18n
 	Vue.prototype.$locale = locale;
+	Vue.prototype.$browser = browser;
```
Comment (ryan-ludwig) [inline src/main.js:69]: Adds this object globally
Screen Shot 2021-02-01 at 4 26 01 PM

**Comment by ryan-ludwig** [comment]
@emuvente @mcstover This pass uses Bowser to parse the user agent on the server and client.
Verified the banner shows on Windows 7 + IE11 with javascript turned on and off. Does not show on Windows 7 + Edge.

File: src/main.js (hunk 1 above)
Comment (ryan-ludwig) [reply src/main.js:69]: Perhaps a different name than $browser? Any thoughts?

Comment (emuvente) [reply src/main.js:69]: I'm ok with `$browser`, but if we were to change it I think `$device` might be a good option as it's an object with all the info about the device being used to view the site, not just the browser

**Review by emuvente — APPROVED** [review]
👍  Nice!

**Review by BoulderBrains — APPROVED** [review]

**Review by mcstover — APPROVED** [review]

**Pull request merged**